src/Generator/ConvertGenerator.php

```diff
@@ -154,13 +157,17 @@ private function createEpub( Book $book ) {
 	}
 
 	private function convert( $epubFileName, $outputFileName ) {
-		$command = array_merge(
-			[ $this->getEbookConvertCommand(), $epubFileName, $outputFileName ],
-			explode( ' ', self::$CONFIG[$this->format]['parameters'] )
-		);
-		$process = new Process( $command );
-		$process->setTimeout( $this->timeout ?? 120 );
-		$process->mustRun();
+		try {
+			$command = array_merge(
+				[ $this->getEbookConvertCommand(), $epubFileName, $outputFileName ],
+				explode( ' ', self::$CONFIG[$this->format]['parameters'] )
+			);
+			$process = new Process( $command );
+			$process->setTimeout( $this->timeout ?? 120 );
+			$process->mustRun();
+		} catch ( RuntimeException $e ) {
```

Comment:
Do we want to catch *all* RuntimeExceptions? I thought we'd be catching ProcessTimedOutException here (as something that's common, and not something we can do much about). Other ones, such as ProcessFailedException, we might still want to treat as 500s, because it could be due to e.g. the disk filling up or something.

Reply:
Good call! I've changed it to just capture just `ProcessTimedOutException`.